backend/src/services/block.service.ts

```diff
@@ -206,16 +207,22 @@ export class BlockService {
     };
   }
 
-  private async getSmartContractProcessingInfo(): Promise<{ processedUntil: number; isProcessing: boolean }> {
+  private async getSmartContractProcessingInfo(): Promise<{ processedUntil: number; isProcessing: boolean; mode: string }> {
     const { smartContractHeight, smartContractCommittedHash } = await this.getAndSetLastSubmittedBlockInfo();
+    const mode = await this.parentChainClient.mode();
     const { timestamp } = await this.parentChainClient.getParentChainBlockBySubnetHash(smartContractCommittedHash);
     let isProcessing = true;
     const timeDiff = new Date().getTime() / 1000 - parseInt(timestamp.toString());
-    if (timeDiff > 60) isProcessing = false;
+    if (mode == 'lite') {
```

Comment:
const isProcessing = timeDiff < 120 || (mode == 'lite' && timeDiff<1000)

Reply:
the line mean is 
only full ,check isProcessing is timeDiff < 120 will be  true , or false
only lite ,check isProcessing is timeDiff < 1000 will be  true ,or false